pytest_fixtures/templatesync_fixtures.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from fauxfactory import gen_string
+
+from robottelo import ssh
+
+
+@pytest.fixture()
+def create_import_export_local_dir():
+    """Creates a local directory on satellite from where the templates will
+        be imported or exported to.
+
+    Also copies example template to that directory for test operations
+
+    :param str dir_name: The directory name which will be created as export
+        or import directory under ```/usr/share/foreman_templates```
+    """
+    dir_name = gen_string('alpha')
+    dir_path = f'/usr/share/foreman_templates/{dir_name}'
+    if ssh.command(f'[ -d {dir_path} ]').return_code == 0:
+        if ssh.command(f'rm -rf {dir_path}').return_code == 1:
+            raise OSError(
+                f'The existing export directory {dir_path} still exists! Please '
+                'remove, recreate it and try again'
+            )
+    if not ssh.command(f'mkdir -p {dir_path}').return_code == 0:
```

Comment:
Please make a single statement of file existence check, deletion and  directory creation.
```suggestion
ssh.command(f'[ -d "{dir_path}" ] && rm -rf "{dir_path} && 'mkdir -p {dir_path} "').return_code == 0